src/integration.test.ts

```diff
@@ -58,65 +89,88 @@ describe("Integration Test", (): void => {
     });
   });
 
-  const mockedExec = async (load: boolean, command: string): Promise<void> => {
-    const stdout = "standard output from Bash command";
-    const stderr = "error output from Bash command";
-    child_process.exec.mockImplementationOnce(
-      (_command: any, _options: any, callback: any): any => {
-        callback(null, { stdout, stderr });
-      }
-    );
-
-    await (load ? docker.loadDockerImages() : docker.saveDockerImages());
-
-    expect(core.getInput).nthCalledWith(1, "key", { required: true });
-    expect(core.info).nthCalledWith<[string]>(1, command);
-    expect(child_process.exec).lastCalledWith(
+  const assertExecBashCommand = (
+    infoCallNum: number,
+    childCallNum: number,
+    command: string
+  ): void => {
+    expect(core.info).nthCalledWith<[string]>(infoCallNum, command);
+    expect(child_process.exec).nthCalledWith(
+      childCallNum,
       command,
       EXEC_OPTIONS,
       expect.anything()
     );
-    expect(core.info).lastCalledWith(stdout);
-    expect(core.error).lastCalledWith(stderr);
+    expect(core.error).lastCalledWith(STDERR);
     expect(core.setFailed).not.toHaveBeenCalled();
+    if (command === LIST_COMMAND) {
+      expect(core.info).nthCalledWith(3, dockerImages.join("\n"));
+    }
```

Comment:
Good point.  I think the most appropriate higher level placement is within `assertLoadDockerImages`.  I moved it there; let me know if you intended otherwise.